src/hotspot/share/cds/filemap.hpp

```diff
@@ -194,6 +194,7 @@ class FileMapHeader: private CDSFileMapHeaderBase {
   bool    _compressed_class_ptrs;                 // save the flag UseCompressedClassPointers
   bool    _use_secondary_supers_table;            // save the flag UseSecondarySupersTable
   size_t  _cloned_vtables_offset;                 // The address of the first cloned vtable
+  size_t  _early_serialized_data_offset;          // Data accessed using {ReadClosure,WriteClosure}::serialize()
```

Comment:
Should we print this field in `FileMapHeader::print()`?

Reply:
I added the print out as you suggested.